TRANSLATE.rst

```diff
@@ -0,0 +1,52 @@
+******************
+Translation Guide 
+******************
+
+.. contents:: 
+
+You can help to translate the Read the Docs Sphinx Theme.
+
+Languages availables:
+
+- Spanish contributed by Leonardo J. Caballero G.
+- Your Language by YOUR NAME.
+
+Installation
+============
+
+For translate the Read the Docs Sphinx Theme you need install the following packages:
+
+.. code:: bash
+
+    $ pip install babel Jinja2
```

Comment:
Jinja2 is a sphinx requirement and should already be present as part of development set up for this package.

Reply:
@agjohnson Any news on this comment?